book/src/faq.md

```diff
@@ -101,6 +88,27 @@ involve converting the methods in question to take `&self` argument instead of `
 
 Please consult [`MessageStream`] documentation for details.
 
+## Why aren't property values updating for my service that doesn't notify changes?
+
+A common issue might arise when using a zbus proxy is that your proxy's property values aren't 
+updating. This is due to zbus' default caching policy, which updates the value of a property only
+when a change is signaled, primarily to minimize latency and optimize client request performance.
+By default, if your service does not emit change notifications, the property values will not
+update accordingly. 
+
+However, you can explicitly bypass this caching mechanism for a client:
+### 1. Client built via `dbus_proxy`
+By adding the `#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation to your proxy
+
+### 2. Client built via `ProxyBuilder`
+`ProxyBuilder::uncached_properties` method to your builder
```

Comment:
this phrase doesn't make sense. Please write a full sentence and end with a `.`.